spec/aruba/processes/in_process_spec.rb

```diff
@@ -28,11 +29,13 @@ def execute!
     end
   end
 
-  subject(:process) do
```

Comment:
I think you can keep this `subject` but just create a different `let` for `main_class` in the `#exit_status` describe block.